src/components/editor/property/InputLookupQA.jsx

```diff
@@ -122,6 +114,13 @@ class InputLookupQA extends Component {
     let authority
     let language
     let subauthority
+    const defaults = defaultValuesFromPropertyTemplate(this.props.propertyTemplate)
+
+    if (defaults.length === 0) {
+      // Property templates do not require defaults but we like to know when this happens
+      console.info(`no defaults defined in property template: ${JSON.stringify(this.props.propertyTemplate)}`)
```

Comment:
i think this could get really noisy.  Maybe make it a debug?  (yes, i know it's what was there, but I'm looking at it now!  ;-)

Reply:
If we want to make a ticket to remove this logging, I'd certainly be supportive.  I don't think that change fits in well with the intent of this PR.

Reply:
I guess I'm more of the school "if I'm looking at it now and thinking about it, might as well change it."  But you're not of that school, and that's ok.  This isn't anything I'd go to the mat for.

Reply:
here's a tiny little PR that makes this change:  https://github.com/LD4P/sinopia_editor/pull/767

Reply:
eh, eslint is complaining about `console.debug`.  i don't particularly care one way or the other about this issue.  happy to address per my comment on the PR, or people can close the PR, whatever.

Reply:
meh, i'm starting to believe we should stop logging this at all.